docs/metadata.md

```diff
@@ -101,6 +101,15 @@ using the Python API.
 .. todo:: Add examples of adding table-level metadata schemas.
 ```
 
+### Defaults
+
+```{eval-rst}
+.. todo:: Add examples of setting default values.
+```
+
+Default values for properties can be specified for only
```

Comment:
This is only the case for JSON codec.

Reply:
Oh, I thought you said it was also required by the struct codec. I misunderstood. But I bunged that in a separate commit, so I can remove that. Sorry for the misunderstanding.